src/main/java/uk/gov/hmcts/reform/hmc/data/ActualAttendeeIndividualDetailEntity.java

```diff
@@ -36,8 +38,16 @@ public class ActualAttendeeIndividualDetailEntity implements Serializable {
     @Column(name = "party_actual_sub_channel_type", nullable = false)
     private String partyActualSubChannelType;
 
+    @Column(name = "created_date_time")
```

Comment:
Since this is repeated everywhere, it'd make sense to introduce a class that all the entities needing it extend, instead of repeating. We could have a `BaseEntity`, for example.